SSL certs being created in data/ dir

@danoli3 is this intended / needed?
We didn't need to generate certs in the past when loading urls in OF. 

<img width="771" alt="Image" src="https://github.com/user-attachments/assets/ffea2a3f-0046-4831-8de8-2e09dec48e07" />

Thanks for the explanation @danoli3 

Just testing on our current Linux Mint ( 21.03 ) system and getting this error with Nightly Build:
So guessing we'll need to get this fixed for 0.12.1 

```
Compiling ../../../libs/openFrameworks/utils/ofUtils.cpp
../../../libs/openFrameworks/utils/ofURLFileLoader.cpp: In member function ‘void ofURLFileLoaderImpl::createSSLCertificate()’:
../../../libs/openFrameworks/utils/ofURLFileLoader.cpp:160:29: error: ‘EVP_PKEY_CTX_new_from_name’ was not declared in this scope
  160 |   EVP_PKEY_CTX * pkey_ctx = EVP_PKEY_CTX_new_from_name(NULL, "RSA", NULL);
```

![Image](https://github.com/user-attachments/assets/b8184551-cff3-4db2-bffd-d858fc7ec45f)

--- a/examples/input_output/imageLoaderWebExample/src/ofApp.cpp
+++ b/examples/input_output/imageLoaderWebExample/src/ofApp.cpp
@@ -61,10 +61,12 @@ void ofApp::exit() {
 
 //--------------------------------------------------------------
 void ofApp::keyPressed(int key){
-	if(loading == false) {
-		img.clear();
-		ofLoadURLAsync("https://openframeworks.cc/about/0.jpg","about");
-		loading = true;
+	if( key == ' '){
+		if(loading == false) {
+			img.clear();
+			ofLoadURLAsync("https://openframeworks.cc/about/0.jpg","about");
+			loading = true;
+		}
 	}
 }
 

--- a/libs/openFrameworks/utils/ofURLFileLoader.cpp
+++ b/libs/openFrameworks/utils/ofURLFileLoader.cpp
@@ -9,22 +9,26 @@ using std::set;
 using std::string;
 
 #if !defined(TARGET_IMPLEMENTS_URL_LOADER)
-#include <curl/curl.h>
-#include "ofThreadChannel.h"
-#include "ofThread.h"
-static bool curlInited = false;
-#define MAX_POSTFIELDS_SIZE (1024 * 1024)
-#if !defined(NO_OPENSSL)
-#include <openssl/evp.h>
-#include <openssl/pem.h>
-#include <openssl/x509.h>
-#include <openssl/x509v3.h>
-#include <openssl/err.h>
-#include <iostream>
-#include <fstream>
-#define CERTIFICATE_FILE "ssl/cacert.pem"
-#define PRIVATE_KEY_FILE "ssl/cacert.key"
-#endif
+	#include <curl/curl.h>
+	#include "ofThreadChannel.h"
+	#include "ofThread.h"
+	static bool curlInited = false;
+
+	#define MAX_POSTFIELDS_SIZE (1024 * 1024)
+
+	#define NO_OPENSSL 1
+	#include <openssl/evp.h>
+	#include <openssl/pem.h>
+	#include <openssl/x509.h>
+	#include <openssl/x509v3.h>
+	#include <openssl/err.h>
+	#include <iostream>
+	#include <fstream>
+
+	#if !defined(NO_OPENSSL)
+		#define CERTIFICATE_FILE "ssl/cacert.pem"
+		#define PRIVATE_KEY_FILE "ssl/cacert.key"
+	#endif
 #endif
 
 int ofHttpRequest::nextID = 0;
@@ -324,11 +328,9 @@ ofHttpResponse ofURLFileLoaderImpl::handleRequest(const ofHttpRequest & request)
 		}
 		curl_easy_setopt(curl.get(), CURLOPT_CAPATH, ofToDataPath(caPath, true).c_str());
 		curl_easy_setopt(curl.get(), CURLOPT_CAINFO, ofToDataPath(caFile, true).c_str());
-		curl_easy_setopt(curl.get(), CURLOPT_SSL_VERIFYPEER, false);
-#else
-		curl_easy_setopt(curl.get(), CURLOPT_SSL_VERIFYPEER, 1L);
 #endif
-		curl_easy_setopt(curl.get(), CURLOPT_SSL_VERIFYHOST, 2);
+		curl_easy_setopt(curl.get(), CURLOPT_SSL_VERIFYPEER, false);
+		curl_easy_setopt(curl.get(), CURLOPT_SSL_VERIFYHOST, 2L);
 	}
 	curl_easy_setopt(curl.get(), CURLOPT_URL, request.url.c_str());
 	curl_easy_setopt(curl.get(), CURLOPT_FOLLOWLOCATION, 1L);

